test/fakefs_test.rb

```diff
@@ -1262,7 +1262,16 @@ def test_should_set_pos_with_tell_method
     :winsize, :winsize=,
     :getch,
     :iflush, :ioflush, :oflush,
-    :pathconf
+    :pathconf,
+    :cursor, :cursor=,
+    :beep, :goto, :pressed?,
+
+    # omit methods from io/nonblock
+    :nonblock, :nonblock=, :nonblock?,
+
+    # omit methods from io/wait
+    :nread, :ready?,
+    :wait, :wait_readable, :wait_writable
```

Comment:
Hmm, it seems that `RealFile` isn’t just the original `File` at this point in the tests – it has gained a lot of extra methods…

I’ll investigate a bit more.